add tips for finding reviewers using @noamross' forum post (#110)
ropensci/dev_guide: softwarereview_editor.Rmd

-   Switch numbered tag `to 3/reviewers-assigned` once reviewers are assigned.
-   Invite authors and reviewers to rOpenSci Slack if they aren't on this Slack already.

#### How to look for reviewers

##### Where to look for reviewers?

As a (guest) editor, use 

> * the potential suggestions made by the submitter(s), 

Reviewer: ```suggestion
* the potential suggestions made by the submitter(s), (although submitters may have a narrow view of the types of expertise needed.  We suggest not using more than one of suggested reviewers)
```

* the Airtable database of reviewers and volunteers 
* and the authors of [rOpenSci packages](https://ropensci.org/packages/). 

When these sources of information are not enough, 
* ping other editors in Slack for ideas, 

> * look for users of the package (via their opening issues in the repository, starring it, citing it in papers, talking about it on Twitter). 

Reviewer: ```suggestion
* look for users of the package or of the data source/upstream service the package connects to (via their opening issues in the repository, starring it, citing it in papers, talking about it on Twitter). 
```

* You can also search for authors of related packages on [r-pkg.org](https://r-pkg.org/).
* R-Ladies has a [directory](http://rladies.org/directory/) specifying skills and interests of people listed.

#### Criteria for choosing a reviewer

Here are criteria to keep in mind when choosing a reviewer. You might need to piece this information together by searching CRAN and the potential reviewer’s GitHub page and general online presence (personal website, Twitter).

* Has not reviewed a package for us within the last 6 months.
* Some package development experience.

> * Some domain experience.

Reviewer: ```suggestion
* Some domain experience in the field of the package or data source
```

* No [conflicts of interest](#coi).
* Try to balance your sense of the potential reviewer’s experience against the complexity of the package.
* Diversity - with two reviewers both shouldn’t be cis white males.

> * Some evidence that they are interested in openness or R community activities although blind emailing is fine.

Reviewer: ```suggestion
* Some evidence that they are interested in openness or R community activities, although blind emailing is fine.
```

> Each submission should be reviewed by _two_ package reviewers. Although it is fine for one of them to have very little package development experience and more domain knowledge, the review should not be split in two, both reviewers need to review the package comprehensively, though from their particular perspective.

Reviewer: ```suggestion
Each submission should be reviewed by _two_ package reviewers. Although it is fine for one of them to have less package development experience and more domain knowledge, the review should not be split in two.  Both reviewers need to review the package comprehensively, though from their particular perspective.  In general, at least one reviewer should have prior reviewing experience, and of course inviting one new reviewer expands our pool of reviewers.
```
